sections/homepage/ChartBanner/ChartBanner.tsx

```diff
@@ -20,7 +19,11 @@ const FAQ = () => {
 	return (
 		<StyledStackSection>
 			<ChartGraphicContainer>
-				<ChartBannerImage src={ChartBanner} alt="" webp={true} />
+				<picture>
+					<source srcSet={`${ChartBanner}?webp`} type="image/webp" />
+					<source srcSet={ChartBanner} type="image/png" />
+					<ChartBannerImage src={ChartBanner} />
+				</picture>
```

Comment:
Long shot, but you could probably create a `Webp` component. That besically takes in the `ChartBanner` and generates the rest. I think there are some other places in this PR where you could reuse it.